pkg/compute/store/kvstore/envelopes.go

```diff
@@ -0,0 +1,38 @@
+package kvstore
+
```

Comment:
I think you forgot to delete this package

Reply:
Was leaving it until I was sure I didn't need it :D